foundations/foundation-config/src/main/java/org/apache/servicecomb/config/archaius/sources/AbstractConfigLoader.java

```diff
@@ -63,7 +63,16 @@ public ConfigModel load(URL url) throws IOException {
     ConfigModel configModel = new ConfigModel();
     configModel.setUrl(url);
     configModel.setConfig(config);
-    Object objOrder = config.get(orderKey);
+
+    Object objOrder = config.get(ORDER_KEY);
+    if (objOrder == null) {
+      // compatible check
+      objOrder = config.get("cse-config-order");
+      if (objOrder != null) {
+        LOGGER.error("cse-config-order will not be supported in future, please change it to servicecomb-config-order");
```

Comment:
I think it should be warning, as the program is not stop working here.

Reply:
This is quick fix for users, so I give error not warning. Users see this, it quite important to fix it immediately. Some others like above duplicate key, I give warning, as users can not change common modules. 
I am thinking of throw an exception here, just like cse.configurationSource.additionalUrls do, but after quite a lot consideration, give an error message is better.